libs/application/templates/children-residence-change/src/dataProviders/MockChildrenNationalRegistryProvider.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Application,
+  BasicDataProvider,
+  FailedDataProviderResult,
+  SuccessfulDataProviderResult,
+} from '@island.is/application/core'
+import { CRCApplication } from '../types'
+
+import { PersonResidenceChange } from '@island.is/application/templates/children-residence-change'
+
+/** This is a temporary mock provider for children residence change. National registry team are setting up real provider which we will connect to once it is up. **/
```

Comment:
Is this comment valid? 
We could say something about the config and how all this works?

Reply:
Going to remove and add some documentation to our readme in a seperate PR when the mock structure is a little more mature